Dockerfile_TNS

```diff
@@ -0,0 +1,11 @@
+FROM scimma/base:0.1.1
+RUN yum -y install git unzip
```

Comment:
You'll want to refresh metadata: `RUN yum -y update && yum -y install git unzip`